share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -811,6 +811,21 @@ deprecated-ci-build:
   tags: [ "cray-sles-zen4", "public" ]
   extends: [ ".generate-cray" ]
 
+.generate-cray-sles-amd:
```

Comment:
These shouldn't be needed, they don't change anything from the existing cray generation. The only reason to add stuff at this level is if generate needs to run on different infrastructure or some other lower level CI decision.

For this, I would wait on the other PR that moves the existing cray stack mirrors and rebase.

Reply:
Oops. Looks like I forgot to update this file when I split the stacks.